src/site/content/en/learn/css/the-cascade/index.md

```diff
@@ -0,0 +1,278 @@
+
+The order in which your CSS rules appear and how they appear is taken into consideration by the cascade,
+ while it calculates conflict resolution.
+
+The demo right at the start of this lesson is the most straightforward example of position.
+There are two rules that have selectors of identical specificity,
+so the last one to be declared won.
+
+Styles can come from various sources on an HTML page,
+such as a `<link>` tag,
+an embedded `<style>` tag,
+ and inline CSS as defined in an element's `style` attribute.
+
+If you have a `<link />` that includes CSS at the top of your HTML page,
+then another `<link />` that includes CSS at the bottom of your page: the bottom `<link />` will have the most specificity.
+The same thing happens with embedded `<style>` elements, too.
+They get more specific, the further down the page they are.
+
+<figure class="w-figure">
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'NWdPaWv'
+} %}
+  <figcaption class="w-figcaption">The button has a blue background,
+  as defined by CSS which is included by a <code>&lt;link /&gt;</code> element.
+  A CSS rule that sets it to be dark is in a second linked stylesheet
+  and is applied because of its later position.</figcaption>
+</figure>
+
+This ordering also applies to embedded `<style>` elements.
+If they are declared before a `<link />`,
+the linked stylesheet's CSS will have the most specificity.
+
+<figure class="w-figure">
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'xxgbLoB'
+} %}
+<figcaption class="w-figcaption">The <code>&lt;style&gt;</code> element is declared in the <code>&lt;head&gt;</code>,
+while the <code>&lt;link /&gt;</code> element is declared in the <code>&lt;body&gt;</code>.
+This means it get more specificity than the <code>&lt;style&gt;</code> element</figcaption>
+</figure>
+
+An inline `style` attribute with CSS declared in it will override all other CSS,
+regardless of its position, unless a declaration has `!important` defined.
+
+Position also applies in the order of your in a CSS rule.
+In this example, the element will have a purple background because `background: purple` was declared last.
+
+```css
+.my-element {
+	background: green;
+	background: purple;
+}
+```
+
+Being able to specify two values for the same property
+can be a simple way to create fallbacks for browsers that do not support a particular value.
+In this next example, `font-size` is declared twice.
+If `clamp()` is supported in the browser,
+then the previous `font-size` declaration will be discarded.
+If `clamp()` isn't supported by the browser,
+the initial declaration will be honored, and the font-size will be 1.5rem
+
+```css
+.my-element {
+	font-size: 1.5rem;
+	font-size: clamp(1.5rem, calc(1rem + 3vw), 2rem);
+}
+```
+
+<figure class="w-figure">
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'xxgbPMP'
+} %}
+</figure>
+
+{% Aside %}
+This approach of declaring the same property twice works because browsers ignore values they don't understand.
```

Comment:
Maybe add a callout how: "Unlike other programming languages, CSS will not throw an error or break your program when it detects a line it cannot parse. Instead, that line is ignored."

Reply:
Yeh I like that